README.md

```diff
@@ -32,7 +34,7 @@ Install all non-ROS dependencies of RMF packages,
 sudo apt update && sudo apt install \
   git cmake python3-vcstool curl \
   qt5-default \
-  python3-shapely python3-yaml python3-requests python3-flask-cors \
+  ignition-edifice \
   -y
 python3 -m pip install flask-socketio
```

Comment:
Can we push `flash_socketio` to upstream `rosdep`? Assuming rosdep will allow us to install the pip version of the package. If I remember correctly, the debian version was a lower version that what is required.

Reply:
From the [PR](https://github.com/ros/rosdistro/pull/29086) to `rosdep`, I did include `python3-flask-socketio` and it is available now. Yes you are right, the issue here is related to the official "ubuntu 20.04" package is using `flask_socketio` version `4.X`, yet our application requires `5.X`. 

The good news here is, the upcoming Ubuntu 21.04 will be using `socketio 5.X`  https://packages.ubuntu.com/source/hirsute/flask-socketio

Reply:
I see, so as of now including `<exec_depend>python3-flask-socketio</exec_depend>` in `package.xml`s will result in rosdep installing `4.X`?

Reply:
Yes, thats right.